.github/workflows/deploy.yml

```diff
@@ -14,3 +14,13 @@ jobs:
       - run: flyctl deploy --remote-only
         env:
           FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
+
+  purge:
```

Comment:
Can you ensure this runs only if the `deploy` step passes?

Reply:
Fixed